### ISSUE TITLE
IndexType mappings/changes

### ISSUE THREAD

**Issue by jseppi** [TNRIS/historical-aerials#5]
Here are the current distinct values in the DB now:

```
"L" ,
"L/P" ,
"LINE" ,
"LM" ,
"LP" ,
"LPM" ,
"M" ,
"Missing" ,
"P" ,
"PL" ,
"PM" ,
"PML" ,
"T" ,
"p"
```

**Comment by jseppi** [comment]
@ThorTheDestroyer ping :dango:

**Comment by jseppi** [comment]
How do you want the current values modified and how would you like this field displayed during Edit/Create in the new app?

**Comment by ThorTheDestroyer** [comment]
L
P
M
LP
LM
MP
 LPM

that should cover all of them. I can't think of any other combinations, right?  Just have it be a drop down with those options.

**Comment by jseppi** [comment]
Ok, what should the existing values "Missing" and "T" be mapped to?

**Comment by ThorTheDestroyer** [comment]
nothing, leave them out

**Comment by ThorTheDestroyer** [comment]
make it an optional value

**Comment by jseppi** [comment]
yes sir :shipit:

**Comment by jseppi** [comment]
closed in 0129759020ca281a2f4201974983e1f49a808bcc
